pkg/controller/dataimportcron-controller_test.go

```diff
@@ -706,6 +706,64 @@ var _ = Describe("All DataImportCron Tests", func() {
 			Entry("has no tag", imageStreamName, 1),
 		)
 
+		It("XXXXXX Should succeed garbage collecting old version DVs", func() {
```

Comment:
Don't need test ID in unit tests (XXXXXX)

Reply:
Fixed